wiki/Katu/id.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - "100"
+  - katsu
+---
+
+# Katu
+
+*Lihat juga: [Geki](/wiki/Geki)*
+
+*Diharapkan tidak keliru dengan Katu dari osu!taiko*
+
+**Katu (喝)**, *Katsu* atau *Beat!*  merupakan istilah dalam [penilaian](/wiki/Score) yang digunakan ketika pemain menyelesaikan suatu comboset tidak dengan [akurasi](/wiki/Gameplay/Accuracy) sempurna.
+Namun, nilai ini tidak dapat diraih apabila mendapatkan 50 atau lebih rendah dalam combo tersebut.
+
+Ada dua jenis Katu, jenis tersebut ditentukan dengan penilaian terakhir dalam akhir comboset.
+Katu yang diakhiri dengan 300 memberikan health bonus yang lebih dibanding dengan Katu yang diakhiri dengan 100, namun keduanya lebih lemah daripada Geki.
```

Comment:
```suggestion
Katu yang diakhiri dengan 300 memberikan bonus health yang lebih dibanding dengan Katu yang diakhiri dengan 100, namun keduanya lebih lemah daripada Geki.
```
This article is in Indonesian, I prefer to swap their positions. `Bonus` can also be found at KBBI.